src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,7 @@
+.catalog {
+  width: 1024px;
```

Comment:
checklist N10
don't hardcode max-width, your catalog container should work fine for any number of cards